packages/core/src/middleware/koa-auth.ts

```diff
@@ -23,6 +33,22 @@ export default function koaAuth<StateT, ContextT>(): Middleware<
         { supportedTypes: [bearerToken] }
       )
     );
+    const jwt = authorization.slice(bearerToken.length + 1);
```

Comment:
Not all apis need to be validated? Some apis can be requested by anonymous users like retrieving configurations about an app.

Reply:
read the pr summary and the code, i've separated into 2 routers